content/techniques/authentication.md

```diff
@@ -22,9 +22,10 @@ First we need to install the required packages. Passport provides a strategy cal
 
 ```bash
 $ npm install --save @nestjs/passport passport passport-local
+$ npm install --save-dev @types/passport-local
 ```
 
-> Warning **Notice** For **any** Passport strategy you choose, you'll always need the `@nestjs/passport` and `passport` packages. Then, you'll need to install the strategy-specific package (e.g., `passport-jwt` or `passport-local`) that implements the particular authentication strategy you are building.
+> Warning **Notice** For **any** Passport strategy you choose, you'll always need the `@nestjs/passport` and `passport` packages. Then, you'll need to install the strategy-specific package (e.g., `passport-jwt` or `passport-local`) that implements the particular authentication strategy you are building. In addition, you can also install the type definitions for `passport-local`, which provide assistance while writing TypeScript code.
```

Comment:
Since this notice is generic with respect to passport strategy, perhaps the wording about types should also be generic?  E.g., "In addition, you can also install the type definitions for any Passport strategy, such as `@types/passport-local` as shown above, which provide assistance while writing TypeScript code."

Reply:
Good point, I'll update the PR

Reply:
Done!